libmscore/scoreElement.h

```diff
@@ -171,6 +171,9 @@ struct ElementName {
 
 class ScoreElement {
       Score* _score;
+#ifdef SCRIPT_INTERFACE
+      QObject* _plugInWrapper;
```

Comment:
You can define a default value for this variable just here:
`QObject* _plugInWrapper = nullptr;`
Then there will be no need in having two different constructors depending on `SCRIPT_INTERFACE` definition and in initializing this value explicitly in copy constructor.

Reply:
Thanks! Much cleaner now.

-Dale